inc/class-command.php

```diff
@@ -139,7 +142,17 @@ protected function init( InputInterface $input, OutputInterface $output ) {
 			return;
 		}
 
-		return $this->run_command( 'vagrant up' );
+		$this->start( $input, $output );
+
+		// And run the initial setup, if the user wants to.
+		$question = new ConfirmationQuestion( 'Install SSL certificate? [Y/n] ', true );
```

Comment:
Minor: Nothing uses `SSL` any more so this should say `security certificate` or maybe `HTTPS certificate`.